guildwars2/account.py

```diff
@@ -347,24 +347,50 @@ def refined_armor_scan(a, b=ids_refined_envoy_armor):
             return await self.error_handler(ctx, e)
 
         def check(item):
```

Comment:
Needs a better function name, it's not really checking anything, rather returning count